src/terrain/routes/webhooks.clj

```diff
@@ -10,4 +10,6 @@
     [config/app-routes-enabled]
 
     (PUT "/webhooks" [:as {:keys [body]}]
-      (service/success-response (apps-client/save-webhooks body)))))
+      (service/success-response (apps-client/save-webhooks body)))
+    (GET "/webhooks/types" [:as {:keys [params]}]
```

Comment:
I suspect you don't need anything inside the `[]` here, since you're not using it below.

Reply:
Ah, good catch.